src/ert/ert3/evaluator/_builder.py

```diff
@@ -156,6 +156,7 @@ def build_ensemble(  # pylint: disable=too-many-arguments
         .set_max_retries(0)
         .set_executor(driver)
         .set_id(ens_id)
+        .set_exp_id("0")
```

Comment:
is this work in progress, or should it always be zero for "ert3"?

Reply:
Had not planned for ert3 yet to be honest. Removed either way, will not be part of `EnsembleBuilder`